### PR TITLE
Adding yieldfrom recipe

### PR DESCRIPTION
**Pull request by ruben-arts**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example). (packed in tarfile)
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ruben-arts** [comment]
@conda-forge/help-python, ready for review

**Comment by ruben-arts** [comment]
He, @wolfv could you give this one a look-see?

File: recipes/yieldfrom/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  imports:
+    - yieldfrom
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/Nurdok/yieldfrom/
+  summary: A backport of the `yield from` semantic from Python 3.x to Python 2.7
```
Comment (wolfv) [inline recipes/yieldfrom/meta.yaml:34]: This seems to contradict the `python >=3.6`? 

But I guess you still need this package as a dependency somewhere?

Comment (ruben-arts) [reply recipes/yieldfrom/meta.yaml:34]: Yes you're completely right. This is a way for us to get it working as we have right now. And refactor it using Conda environments.

Comment (wolfv) [reply recipes/yieldfrom/meta.yaml:34]: How about you make the requirement into `python >=2.7` then?

Comment (ruben-arts) [reply recipes/yieldfrom/meta.yaml:34]: Done :heavy_check_mark:

**Pull request merged**